dist/rzslider.js

```diff
@@ -1268,6 +1269,20 @@
           this.scope.barStyle = {
             backgroundColor: color
           };
+        } else if (this.options.selectionBarGradient) {
+          var offset = this.options.showSelectionBarFromValue !== null ? this.valueToPosition(this.options.showSelectionBarFromValue) : 0,
+            reversed = offset - position > 0 ^ isSelectionBarFromRight,
```

Comment:
It's a XOR. It means either `offset - position > 0` or `isSelectionBarFromRight` is true, but not both.